back/app/services/projects_finder_service.rb

```diff
@@ -81,59 +81,39 @@ def projects_with_active_phase(projects)
   # ordered by the follow created_at (most recent first).
   # => [Project]
   def followed_by_user
-    subquery = @projects
-      .joins('INNER JOIN admin_publications AS admin_publications ON admin_publications.publication_id = projects.id')
-      .where(admin_publications: { publication_status: 'published' })
-      .joins(
-        'LEFT JOIN followers AS project_followers ON project_followers.followable_id = projects.id ' \
-        'AND project_followers.followable_type = \'Project\''
-      )
-      .joins('LEFT JOIN ideas ON ideas.project_id = projects.id')
-      .joins(
-        'LEFT JOIN followers AS idea_followers ON idea_followers.followable_id = ideas.id ' \
-        'AND idea_followers.followable_type = \'Idea\''
-      )
+    subquery = Follower
+      .where(user_id: @user.id)
       .joins(
-        'LEFT JOIN areas_projects ON areas_projects.project_id = projects.id'
+        'LEFT JOIN areas AS followed_areas ON followers.followable_type = \'Area\' ' \
+        'AND followed_areas.id = followers.followable_id'
       )
+      .joins('LEFT JOIN areas_projects ON areas_projects.area_id = followed_areas.id')
       .joins(
-        'LEFT JOIN followers AS area_followers ON area_followers.followable_id = areas_projects.area_id ' \
-        'AND area_followers.followable_type = \'Area\''
+        'LEFT JOIN topics AS followed_topics ON followers.followable_type = \'Topic\' ' \
+        'AND followed_topics.id = followers.followable_id'
       )
+      .joins('LEFT JOIN projects_topics ON projects_topics.topic_id = followed_topics.id')
       .joins(
-        'LEFT JOIN projects_topics ON projects_topics.project_id = projects.id'
+        'LEFT JOIN ideas AS followed_ideas ON followers.followable_type = \'Idea\' ' \
+        'AND followed_ideas.id = followers.followable_id'
       )
       .joins(
-        'LEFT JOIN followers AS topic_followers ON topic_followers.followable_id = projects_topics.topic_id ' \
-        'AND topic_followers.followable_type = \'Topic\''
-      )
-      .where(
-        'project_followers.user_id = :user_id OR idea_followers.user_id = :user_id ' \
-        'OR area_followers.user_id = :user_id OR topic_followers.user_id = :user_id',
-        user_id: @user.id
-      )
-      .select(
-        'projects.id AS project_id, ' \
-        'MAX(GREATEST(' \
-        'project_followers.created_at, ' \
-        'idea_followers.created_at, ' \
-        'area_followers.created_at, ' \
-        'topic_followers.created_at' \
-        ')) AS greatest_created_at'
+        'LEFT JOIN projects ON ' \
```

Comment:
Given you always want to return projects, you can probably do an `INNER JOIN` here. Probably won't impact performance - I've always thought inner joins were more efficient, but the internet is very inconclusive on this!